NineChronicles.Headless/GraphTypes/States/WorldBossKillRewardRecordType.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using GraphQL.Types;
+using Nekoyume.Model.State;
+
+namespace NineChronicles.Headless.GraphTypes.States
+{
+    public class WorldBossKillRewardRecordType : ObjectGraphType<WorldBossKillRewardRecord>
+    {
+        private class WorldBossKillRewardRecordMapType : ObjectGraphType<KeyValuePair<int, bool>>
+        {
+            public WorldBossKillRewardRecordMapType()
+            {
+                Field<IntGraphType>("bossLevel", resolve: context => context.Source.Key);
+                Field<NonNullGraphType<BooleanGraphType>>(
+                    "claimed",
+                    description: "check reward already claimed. if already claimed return true.",
+                    resolve: context => context.Source.Value);
+            }
+        }
+
+        public WorldBossKillRewardRecordType()
+        {
+            Field<NonNullGraphType<ListGraphType<WorldBossKillRewardRecordMapType>>>(
```

Comment:
I think the element's type is also non-nullable because it doesn't return like:

```
{
  map: [
    {
      bossLevel: 0,
      claimed: false
    },
    null,
    {
      bossLevel: 3
      claimed: true
    }
  ]
}
```

```suggestion
            Field<NonNullGraphType<ListGraphType<NonNullGraphType<WorldBossKillRewardRecordMapType>>>>(
```